turbo.json

```diff
@@ -7,13 +7,8 @@
       "outputs": ["dist/**", ".next/**"]
     },
     "@skillrecordings/config#build": {
-      "dependsOn": [
-        "^build",
-        "generate",
-        "$CONVERTKIT_API_SECRET",
-        "$NEXT_PUBLIC_CONVERTKIT_TOKEN",
-        "$NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM"
```

Comment:
As of turbo 1.4+, framework and tool specific env vars are largely
handled for you in terms of being task dependencies. These no longer
need to be manually specified.

Docs: https://turborepo.org/docs/core-concepts/caching#automatic-environment-variable-inclusion